webview/src/experiments/components/table/styles.module.scss

```diff
@@ -559,11 +555,12 @@ $bullet-size: calc(var(--design-unit) * 4px);
 }
 
 .experimentCell .rowActions .starSwitch svg {
```

Comment:
How about leaving the filled icon color as is and using the checkbox-border for the unfilled one?

```
.experimentCell .rowActions .starSwitch svg {
  fill: $checkbox-border;
}

.experimentCell .rowActions .starSwitch[aria-checked='true'] svg {
  fill: $row-action-icon-color;
}
```

https://user-images.githubusercontent.com/37993418/196055811-3ba77c37-29a2-4f72-a4da-eec211f5d148.mov

IMO the yellow looks out of place wrt the rest of the UI.

Reply:
Yep, I was trying that alternative also.

I think the star action itself is not as visible enough then. I admit that I like it visually more, but I think we are sacrificing usability to aesthetics in that case.

> IMO the yellow looks out of place wrt the rest of the UI.

Yes, but it's super common UX pattern + if we keep star color a bit more aggressive, we have to do fill color more aggressive as well.

Reply:
> I admit that I like it visually more, but I think we are sacrificing usability to aesthetics in that case.

My opinion is that both are just as usable as one another but one looks better. 

I am not prepared to fight to the death about it.